config/default/rbac/rbac_role.yaml

```diff
@@ -97,3 +97,9 @@ rules:
   - create
   - update
   - patch
+- apiGroups:
```

Comment:
@phoracek btw I tried to add this via kubebuilder command such as was done in [webhook](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/8f197a045e5e092550677df8794271a8708e8a66/pkg/webhook/webhook.go#L48):
`// +kubebuilder:rbac:groups="",resources=namespaces,verbs=get`
it didn't realy work, but would you prefer I insist on it?

Reply:
You say you have added this manually? Does not it get overwritten by generate?

Reply:
generate doesn't do anything to it, so I added it manually.. not sure what I'm doing wrong

Reply:
I'm ok with that.

Reply:
ok I'll leave it without the kubebuilder remark..